JS/Lesson32/onur/my-app/src/components/RecipeList/index.jsx

```diff
@@ -1,12 +1,31 @@
-export const RecipeList = ({recipes}) => {
+import { useState } from "react";
+import { RecipeDetail } from "../RecipeDetail";
+import './styles.css'
+
+
+export const RecipeList = ({ recipes }) => {
+
+	const [recipe, setRecipe] = useState(false);
+	const [SelectedRecipe, setSelectedRecipe] = useState(null);
```

Comment:
Don't forget to use camelCase for naming state values in React
```suggestion
	const [selectedRecipe, setSelectedRecipe] = useState(null);
```